pontoon/base/migrations/0103_bug_1398108_rename_groups_with_duplicate_words.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-09-08 11:37
+from __future__ import unicode_literals
+
+from bulk_update.helper import bulk_update
+
+from django.db import migrations
+from django.db.models import F
+
+
+def rename_groups_with_duplicate_words(apps, schema_editor):
+    Group = apps.get_model('auth', 'Group')
+    groups_to_update = []
+
+    for name in ('managers', 'translators'):
+        double = '{} {}'.format(name, name)
+        for group in Group.objects.filter(name__contains=double):
+            group.name = group.name.replace(double, name)
```

Comment:
I think you could try to use  `.update()` here.

Reply:
I tried that in combination with the `F()` expression, but failed to find a solution for search and replace.

Reply:
https://gist.github.com/simonw/30d2e626772e84edd688ba717a766c89

Reply:
Thanks! Updated.